src/main/kotlin/lotto/model/Store.kt

```diff
@@ -0,0 +1,26 @@
+package lotto.model
+
+import kotlin.math.floor
+
+class Store(private val buyer: Buyer) {
+
+    fun drawLottoNumber(lottoNumbers: List<Int>): List<Prize> {
+        val results = mutableListOf<Prize>()
+        for (lotto in buyer.purchasedLottos) {
+            val matchCount = lotto.numbers.count { lottoNumbers.contains(it) }
+            results.add(Prize.findByMatchCount(matchCount))
+        }
```

Comment:
여긴 for loop가 아니라 아래와 같이 map을 이용해 간단하게 처리가 가능할 것 같네요!

```suggestion
buyer.purchasedLottos.map {
    Prize.findByMatchCount(it.numbers.count { lottoNumbers.contains(it) })
}
```

Reply:
수정하였습니다.
[]